daprdocs/content/en/operations/hosting/kubernetes/kubernetes-dapr-shared.md

```diff
@@ -0,0 +1,86 @@
+---
+type: docs
+title: "Deploy Dapr per-node or per-cluster with Dapr Shared"
+linkTitle: "Dapr Shared"
+weight: 50000
+description: "Learn more about using Dapr Shared as an alternative deployment to sidecars"
+
+---
+
+Dapr automatically injects a sidecar to enable the Dapr APIs for your applications for the best availability and reliability. 
+
+Dapr Shared enables two alternative deployment strategies to create Dapr applications using a Kubernetes `Daemonset` for a per-node deployment or a `Deployment` for a per-cluster deployment. 
+
+- **`DaemonSet`:** When specifying to run Dapr as a Kubernetes `DaemonSet` resource, the daprd container runs on each Kubernetes node. This can reduce network hops between the applications and Dapr. 
+- **`Deployment`:** When running Dapr Shared as a Kubernetes `Deployment`, the Kubernetes scheduler decides in which node the Dapr Shared instance will run.
```

Comment:
@msfussell Same here, I wouldn't change this as this will confuse the main target audience that is already familiar with how Deployments work. I am ok with all the other changes suggested here.